src/components/menu/menu-interim-element.js

```diff
@@ -82,43 +150,36 @@ function MenuProvider($$interimElementProvider) {
           isRemoved: false,
           target: angular.element(opts.target), //make sure it's not a naked dom node
           parent: angular.element(opts.parent),
-          menuContentEl: angular.element(element[0].querySelector('md-menu-content')),
-          backdrop: opts.hasBackdrop && $mdUtil.createBackdrop(scope, "md-menu-backdrop md-click-catcher")
+          menuContentEl: angular.element(element[0].querySelector('md-menu-content'))
         });
       }
 
-      /** Wireup various resize listeners for screen changes */
-      function handleResizing() {
-        opts.resizeFn = function() {
-          positionMenu(element, opts);
-        };
-        angular.element($window).on('resize', opts.resizeFn);
-        angular.element($window).on('orientationchange', opts.resizeFn);
-      }
-
       /**
-       * Place the menu into the DOM and call positioning related functions
+       * Configure various resize listeners for screen changes
        */
-      function showMenu() {
-        opts.parent.append(element);
+      function activateResizing() {
 
-        element.removeClass('md-leave');
-        // Kick off our animation/positioning but first, wait a few frames
-        // so all of our computed positions/sizes are accurate
-        $$rAF(function() {
-          $$rAF(function() {
-            positionMenu(element, opts);
-            // Wait a frame before fading in menu (md-active) so that we don't trigger
-            // transitions on the menu position changing
-            $$rAF(function() {
-              element.addClass('md-active');
-              opts.alreadyOpen = true;
-              element[0].style[$mdConstant.CSS.TRANSFORM] = '';
-            });
+        var debouncedOnResize = (function(target,options){
```

Comment:
`debouncedOnResize` => `repositionMenu` ?
(that fact that it's debounced doesn't really need to be captured in the name)